examples/organiq-raspberry/pi-device.js

```diff
@@ -0,0 +1,74 @@
+
+//////////////////////////////////////////////////////////////////////////////
+//pi-device.js
+//////////////////////////////////////////////////////////////////////////////
+var organiq = require('organiq');
+var Gpio = require('onoff').Gpio;
+var led = new Gpio(14, 'out');
+var  button = new Gpio(4, 'in', 'both');
+//////////////////////////////////////////////////////////////////////////////
+var exit = function () {
+  led.unexport();
+  button.unexport();
+  process.exit();
+}
+process.on('SIGINT', exit);//when interrupted, exit properly 
+//////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////
+var _pressed = 'false';

```

Comment:
// Use strings for 'true' and 'false' to work around problem with 
// Booleans in Organiq